docs/source/contribute/stablehlo.md

```diff
@@ -1,13 +1,9 @@
-Torch Export to StableHLO
---------------------------
+# Torch Export to StableHLO
```

Comment:
this should be in a separate feature dir, this is not a contribute doc.

Reply:
In fact only
```
[Bazel in Pytorch/XLA](https://github.com/pytorch/xla/pull/7988/files#)
[Codegen migration Guide](https://github.com/pytorch/xla/pull/7988/codegen_migration.html)
[OP Lowering Guide](https://github.com/pytorch/xla/pull/7988/op_lowering.html)
[Custom Hardware Plugins](https://github.com/pytorch/xla/pull/7988/plugins.html)
```

belong to `Contribute to Pytorch/XLA`. Please move everything else in a feature dir.

Reply:
A directory named "feature" doesn't fit with the rest of the directory names. They each indicate some sort of action the reader is taking: learning about PT/XLA, contributing to PT/XLA, etc.  The topics currently under "contribute" are only useful for those customers that will be contributing to the PT/XLA project, so I think it makes sense to put them there. @will-cromar had suggested I place them there as well. Please explain why you feel these topics don't belong in "contribute".

Reply:
IMO contribute are a set of docs for the contributor, it can includes things like `how to setup developer env`, `how to contribute a codegen/op lowering`.

Export on the other hand is a feature that we want user to use. It is used to exporting a model to a stableHLO. Contributor does not need to know how export works, but we want user to see this doc and aware this feature exists. sample arguments applied to other docs. Some of the features are for the perfomrances, some features don't but they are still user facing doc, not contributor facing.